masking/mask_terms.py

```diff
@@ -16,97 +15,186 @@ def is_comment_or_empty(s: str) -> str:
     return s.startswith('#') or re.search(r'^\s*$', s)
 
 
-def get_mask(label, index: Optional[int] = None):
-    """
-    Produces a maybe-indexed mask.
-    """
-    if index is None:
-        return '__{}__'.format(label)
-    else:
-        return '__{},{}__'.format(label, index)
+#def get_mask(label, index: Optional[int] = None):
+#    """
+#    Produces a maybe-indexed mask.
+#    """
+#    if index is None:
+#        return '__{}__'.format(label)
+#    else:
+#        return '__{},{}__'.format(label, index)
 
 def singlespace(s: str) -> str:
     if s is not None:
+        #s = regex.sub(r' +', ' ', s.strip())
         s = re.sub(r' +', ' ', s.strip())
     return s
 
 
 class TermMasker:
     def __init__(self,
                  pattern_files: List[str],
-                 add_index: Optional[bool] = False) -> None:
+                 dict_files: List[str],
```

Comment:
probably both `dict_files` and `pattern_files` should default to `[]`:

    pattern_files: List[str] = []
    dict_files: List[str] = []

Since either can be optional.